src/site/content/en/blog/conversion-measurement/index.md

```diff
@@ -0,0 +1,696 @@
+
+<div class="w-table-wrapper">
+  <table class="w-table--top-align">
+    <thead>
+      <tr>
+        <th><code>impressionexpiry</code></th>
+        <th>Depending on conversion time, a conversion report is sent (if the browser is open)...</th>
+        <th>Number of reporting windows</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>30 days, the default and maximum value</td>
+        <td>
+          <ul>
+            <li>2 days after the ad was clicked</li>
+            <li>or 7 days after ad click</li>
+            <li>or <code>impressionexpiry</code> = 30 days after ad click.</li>
+          </ul>
+        </td>
+        <td>3</td>
+      </tr>
+      <tr>
+        <td><code>impressionexpiry</code> is between 7 and 30 days</td>
+        <td>
+          <ul>
+            <li>2 days after ad click</li>
+            <li>or 7 days after ad click</li>
+            <li>or <code>impressionexpiry</code> after ad click.</li>
+          </ul>
+        </td>
+        <td>3</td>
+      </tr>
+      <tr>
+        <td><code>impressionexpiry</code> is between 2 and 7 days</td>
+        <td>
+          <ul>
+            <li>2 days after ad click</li>
+            <li>or <code>impressionexpiry</code> after ad click.</li>
+          </ul>
+        </td>
+        <td>2</td>
+      </tr>
+      <tr>
+        <td><code>impressionexpiry</code> is under 2 days</td>
+        <td>
+            2 days after ad click
+        </td>
+        <td>1</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<figure class="w-figure">
+  <img src="./report-timing.jpg" alt="Chronology of what reports are sent when">
+</figure>
+
+See [Sending Scheduled
+Reports](https://github.com/WICG/conversion-measurement-api#sending-scheduled-reports) for more
+details on timing.
+
+## Example
+
+{% Banner 'info', 'body' %} To see this in action, try out the
+[demo](https://goo.gle/demo-event-level-conversion-measurement-api) ⚡️ and see the corresponding
+[code](https://github.com/GoogleChromeLabs/trust-safety-demo/tree/main/conversion-measurement). {%
+endBanner %}
+
+Here's how the API records and reports a conversion. Note that this is how a click-to-convert flow
+would work with the current API. Future iterations of this API [may be different](#use-cases).
+
+### Ad click (steps 1 to 5)
+
+<figure class="w-figure">
+  <img src="./steps-1.jpg" alt="Diagram: ad click and click storage">
+</figure>
+
+An `<a>` ad element is loaded on a publisher site by `adtech.example` within an iframe.
+
+The adtech platform developers have configured the `<a>` element with conversion measurement
+attributes:
+
+```html
+<a
+  id="ad"
+  impressiondata="776f09351f5809c5"
+  conversiondestination="https://advertiser.example"
+  reportingorigin="https://adtech.example"
+  impressionexpiry="864000000"
+  href="https://advertiser.example/shoes07"
+  target="_blank"
+>
+  <img src="/images/shoe.jpg" alt="shoe" />
+</a>
+```
+
+This code specifies the following:
+
+{% Aside %}
```

Comment:
@maudnals FYI you were missing a huge table here

![image](https://user-images.githubusercontent.com/4713486/95237460-1952c400-07bd-11eb-83ea-3bf071fe5525.png)